test/unit/argument_parser/format_parse_test.cpp

```diff
@@ -14,7 +14,7 @@ TEST(parse_type_test, add_option_short_id)
     std::string option_value;
 
     const char * argv[] = {"./argument_parser_test", "-s", "option_string"};
-    seqan3::argument_parser parser{"test_parser", 3, argv, false};
+    seqan3::argument_parser parser{"test_parser", 3, argv, seqan3::update_notifications::disabled};
```

Comment:
Does these test have a test fixture? If so you could provide a functio; `make_parser_without_updates(args...)` which always creates a parser with the arguments but the update notification set to `off`.

Reply:
No this test does not have a test fixture yet. I would avoid restructuring the whole test now but I can make a follow up issue?

Reply:
Well, yes. I mean the whole point of incremental refactoring is to do it, when you are woking on it. You can make a follow up issue but that shoud be attended right away anyway. It is ok to have this PR merged before though, but I suggest handling this right away would be the best choice.